{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/base.py

```diff
@@ -404,11 +404,6 @@
                     'selector': 'h1,h2,h3,h4,h5,h6',
                     'classes': 'wys-Title-large'
                 },
-                {
-                    'title': 'Extra large',
-                    'selector': 'h1,h2,h3,h4,h5,h6',
-                    'classes': 'wys-Title-extraLarge'
-                }
             ]
```

Comment:
Missing the option to select `wys-Standfirst`

Reply:
My provisional solution to this is to remove the standfirst test in the frontend template, as not all sites use a standfirst style.